packages/@okta/i18n/package.json

```diff
@@ -87,19 +87,19 @@
   },
   "_requested": {
     "name": "@okta/i18n",
-    "raw": "@okta/i18n@37.4.0",
-    "rawSpec": "37.4.0",
+    "raw": "@okta/i18n@40.2.0",
+    "rawSpec": "40.2.0",
     "scope": "@okta",
-    "spec": "37.4.0",
+    "spec": "40.2.0",
     "type": "version"
   },
   "_requiredBy": [
-    "/"
+    "#USER"
```

Comment:
mind changing this to ```/``` please? I update the [i18n wik](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/238007322/Localization+Dev#Localization(Dev)-Inokta-signin-widget)i with that step

Reply:
Sorry! Forgot that step